packages/fast-sketch-library/readme.md

```diff
@@ -0,0 +1,40 @@
+# FAST-DNA sketch library
+`@microsoft/fast-sketch-library` is a tool for quickly extracting sketch libraries from a website. It uses [https://github.com/brainly/html-sketchapp/](https://github.com/brainly/html-sketchapp/) to create an "almost sketch" JSON file which can be loaded into sketch using the plugin provided by `html-sketchapp`.
+
+## Usage
+Install the package
+`npm i -D @microsoft/fast-sketch-library`
+
+Set up a node script to extract symbols from a library
+
+```js
+const extractSymbolLibrary = require("@microsoft/fast-sketch-library").extractSymbolLibrary;
+const fs = require("fs");
+
+const config = {
+    name: "Fast-dna MSFT design kit",
+    pageWidth: 1600,
+    pageHeight: 1600,
+    sources: [
+        {
+            url: "http://your-website.com",
+            selectors: ".some-element-selector"
+        }
```

Comment:
if you determine in other comments a change is needed here then we should update this to match.

Reply:
This is the readme - I don't think it needs to reflect our actual implementation and I think it's actually better if it is generic.

Reply:
This was generic about the overall code review not the readme. But, i get it.